runtime/near-vm-logic/src/logic.rs

```diff
@@ -2389,6 +2389,22 @@ impl<'a> VMLogic<'a> {
         Ok(res? as u64)
     }
 
+    /// Debug print given utf-8 string to node log. It's only available in Sandbox node
+    ///
+    /// # Errors
+    ///
+    /// * If string is not UTF-8 returns `BadUtf8`
+    ///
+    /// # Cost
+    ///
+    /// 0
```

Comment:
I don't think this is correct, `get_utf8_string` costs some gas.

Reply:
Fixed. Use a sandbox version of get utf8 string. The idea is we still want this cost 0, so adding debug log to a function won't cause run out of gas for an existing funcall.